docs/.vuepress/theme/components/Home.vue

```diff
@@ -15,7 +15,7 @@
       <div class="copy">
         <div class="text">
           <h2>
-            <span class="buidler">Buidler</span> is a task runner for Ethereum
+            <span class="hardhat">Hardhat</span> is a task runner for Ethereum
```

Comment:
@alcuadrado in another part you changed "task runner" for "development environment". Should that be done here too?

Reply:
We are going to do that in the entire site in a few days. In this pr I just removed all the unnecessary mentions of buidler, cause I was getting some vuepress errors and warnings.